docs/assets/css/styleguide.scss

```diff
@@ -431,18 +431,17 @@ h3 + .button_wrapper {
 
 .button_wrapper {
   clear: both;
-  display: table;
+  display: block;
   margin-left: -1rem;
   padding: 0rem 1rem;
 
-  &:after {
-    content:"\A";
-    white-space:pre;
-  }
-
   button:last-child {
     margin-right: 0;
   }
+
+  @include media($small-screen) {
+    display: table;
+  }
```

Comment:
Our coding styleguide places media queries directly after the declaration list (See: "Order" https://pages.18f.gov/frontend/css-coding-styleguide/format/). This media `@include` is treated as a media query. Can you move this after the declaration list and before the nested `button:last-child` element?